packages/fxa-auth-server/config/index.ts

```diff
@@ -842,24 +842,38 @@ const conf = convict({
       },
     },
     playApiServiceAccount: {
+      credentials: {
+        client_email: {
+          default: 'test@localtest.com',
+          doc: 'The email of the service account to use for Play API calls',
+          env: 'SUBSCRIPTIONS_PLAY_CLIENT_EMAIL_CREDENTIAL',
+          format: String,
+        },
+        private_key: {
+          default: '',
+          doc: 'The private key of the service account to use for Play API calls',
+          env: 'SUBSCRIPTIONS_PLAY_PRIVATE_KEY_CREDENTIAL',
+          format: String,
+        },
+      },
+      keyFilename: {
+        default: path.resolve(__dirname, 'secret-key.json'),
```

Comment:
is this the default we want? this is also the key we use to sign fxa assertions with

Reply:
I dn't think it is, no. Replacing with empty string.